bundle/metadata/annotations.yaml

```diff
@@ -4,9 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: splunk-operator
-  operators.operatorframework.io.bundle.channels.v1: "beta,stable"
-  operators.operatorframework.io.bundle.channel.default.v1: "stable"
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.21.0
+  operators.operatorframework.io.bundle.channels.v1: alpha
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.15.0
```

Comment:
is this 1.15 or 1.18? @vivekr-splunk ?

Reply:
These values are coming form develop
https://github.com/splunk/splunk-operator/blob/develop/bundle/metadata/annotations.yaml#L7-L8
If we need updates I will push changes on develop first.